### DISCUSSION TITLE
Q: 部署网页无法打开 || Q: The deployment webpage cannot be opened

### DISCUSSION THREAD

**Discussion by YOU-MING-6** [walinejs/waline#2639] [Q&A] [answered]
### Describe the bug | 问题描述

我用cloudflare部署了waline评论，开始部署后的网页打不开怎么办
![1722494372525](https://github.com/user-attachments/assets/ab8e1412-6d50-4d92-8125-55f5c8e9620f)

### Website URL | 问题网站

https://waline.youming.dns.army/

### Where your waline deploy? | 服务部署在哪里？

Other

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I deployed waline comments using cloudflare. What should I do if the web page cannot be opened after deployment?
![1722494372525](https://github.com/user-attachments/assets/ab8e1412-6d50-4d92-8125-55f5c8e9620f)

### Website URL | Problem website

https://waline.youming.dns.army/

### Where your waline deploy? | Where is the service deployed?

Other

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
https://waline.youming.dns.army/ 地址有问题，打不开

**Reply by YOU-MING-6** [reply]
我这边放弃使用cloudflare部署了。
问一下，Vercel 怎么自定义域名，我这边总是失败
![image](https://github.com/user-attachments/assets/bc14a9d1-ea2b-4531-8266-821292acb49c)
![image](https://github.com/user-attachments/assets/0b0265ca-e4aa-4415-b55b-c098a75b1612)


---
I gave up using cloudflare for deployment.
I would like to ask, how do I customize a domain name in Vercel? It always fails on my end.
![image](https://github.com/user-attachments/assets/bc14a9d1-ea2b-4531-8266-821292acb49c)
![image](https://github.com/user-attachments/assets/0b0265ca-e4aa-4415-b55b-c098a75b1612)
